src/lib/components/BackToTop.svelte

```diff
@@ -0,0 +1,34 @@
+<script lang="ts">
+	import { slide } from 'svelte/transition';
+	import ArrowIcon from '$lib/icons/arrow.svg?component';
+	let y: any;
+</script>
+
+<svelte:window bind:scrollY={y} />
+{#if y >= 80}
+	<a href="/#" transition:slide>
```

Comment:
De transition directive is nice, maar zou je hem misschien kunnen aanpassen zodat ie er iets cleaner uit ziet? Misschien een fade, fly of scale gebruiken, of slide aanpassen.  Naar mijn gevoel wordt de pijl nogal gesquished in de out animation. Je zou ook een andere in en uit animatie kunnen doen. 

Vind je dit een goed idee, lmk.